app/controllers/idv_controller.rb

```diff
@@ -1,7 +1,30 @@
 class IdvController < ApplicationController
-  def p1
+  def register_user(args = nil)
+    agent = Proofer::Agent.new(vendor: :mock)
+    app_vars = params.slice(:first_name, :last_name, :dob)
+    applicant = Proofer::Applicant.new(app_vars)
+    session[:resolution] = agent.start(applicant)
+    session[:question_number] = 0
+    render 'question', locals: {quest: session[:resolution].questions[session[:question_number]]}
   end
```

Comment:
We're still gathering info about the various ways the vendors implement their processes. I'd say we could hold off major architectural decisions until we have more info about how they work, unless @pkarman feels we have enough info to make those calls